src/objective-c/tests/run_plugin_option_tests.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Copyright 2015 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Don't run this script standalone. Instead, run from the repository root:
+# ./tools/run_tests/run_tests.py -l objc
+
+set -ev
+
+cd $(dirname $0)
+
+# Run the tests server.
+
+ROOT_DIR=../../..
+BAZEL=$ROOT_DIR/tools/bazel
+PROTOC=$ROOT_DIR/bazel-bin/external/com_google_protobuf/protoc
+PLUGIN=$ROOT_DIR/bazel-bin/src/compiler/grpc_objective_c_plugin
+RUNTIME_IMPORT_PREFIX=prefix/dir/
+
+[ -f $PROTOC ] && [ -f $PLUGIN ] || {
+    BAZEL build @com_google_protobuf//:protoc //src/compiler:grpc_objective_c_plugin
+}
+
+rm -rf RemoteTestClient/*pb*
+
+$PROTOC \
+    --plugin=protoc-gen-grpc=$PLUGIN \
+    --objc_out=RemoteTestClient \
+    --grpc_out=runtime_import_prefix=$RUNTIME_IMPORT_PREFIX:RemoteTestClient \
+    -I $ROOT_DIR \
+    -I ../../../third_party/protobuf/src \
+    $ROOT_DIR/src/objective-c/examples/RemoteTestClient/*.proto
+
+# Verify the output proto filename
+[ -e ./RemoteTestClient/src/objective-c/examples/RemoteTestClient/Test.pbrpc.m ] || {
+    echo >&2 "protoc outputs wrong filename."
+    exit 1
+}
+
+# Verify paths of protobuf WKTs in generated code contain runtime import prefix.
+[ "`cat RemoteTestClient/src/objective-c/examples/RemoteTestClient/Test.pbrpc.m |
+    egrep '#import ".*\.pbobjc\.h"$' |
```

Comment:
The test condition is a little bit vague here: we want to test that all the WKTs contain runtime import prefix, but not the local imports (CMIIW). The two egrep sentences here appear to be filtering "lines with #import.*prefix/dir/.*pbobjc\.h" exists. That is a little bit too general I think. Since we know that test.proto is importing `empty.proto`, maybe check `#import.*/prefix/dir/GPBEmpty\.pbobjc\.h` and #import "Messages\.pbobjc\.h"?

Also, let's have another sentence to verify the same thing in header files as well.

Reply:
Done. In .h header files, no other proto files will be imported, so I guess we dont need to verify those header files.